project/interface.go

```diff
@@ -37,6 +37,7 @@ type APIProject interface {
 	CreateService(name string) (Service, error)
 	AddConfig(name string, config *config.ServiceConfig) error
 	Load(bytes []byte) error
+	ListStoppedContainers(ctx context.Context, services ...string) ([]string, error)
```

Comment:
Why not, but if we merge as-is, the follow-up have to be in 0.3.0 (as it will update the signature 👼). I can take care of it :stuck_out_tongue_closed_eyes: